docs/_component/editor.client.js

```diff
@@ -119,6 +119,17 @@ export const Editor = ({children}) => {
   )
   const stats = state.file ? statistics(state.file) : {}
 
+  // Create a preview component that can handle errors with try-catch block; for catching invalid JS expressions errors
+  const Preview = useCallback(() => {
+    try {
+      return state.file.result();
```

Comment:
As this component (`state.file.result`) is now wrapped in another component (`Preview`), the benefit if calling it ourselves (`state.file.result()`) instead of using it like any other component (`<state.file.result />`) is lost.
Due to this case, I think it’s clearer to use JSX syntax here:  `<state.file.result />`